src/main/java/com/juniorstart/juniorstart/service/UserService.java

```diff
@@ -51,9 +48,13 @@ public UserService(UserDao userDao, MailService mailService) {
      * @return a current user credentials
      * @throws ResourceNotFoundException if user not find
      */
-    public User getCurrentUser(@CurrentUser UserPrincipal userPrincipal) {
-        return userDao.findByPrivateId(userPrincipal.getId())
+    public UserSummary getCurrentUser(@CurrentUser UserPrincipal userPrincipal) {
```

Comment:
@CurrentUser  w kontrolerze

Reply:
nie rozumiem, co z nim nie tak?

Reply:
@GetMapping("/user/me")
    @PreAuthorize("hasRole('USER')")
    public UserSummary getCurrentUser(@CurrentUser UserPrincipal userPrincipal) {
        return userService.getCurrentUser(userPrincipal);
    }

Reply:
tutaj go używasz do tej samej metody, czyli powtarzasz to samo czyli program 2 razy sprawdza to samo.

Reply:
Nie poprawione